cmd/bb_copy/BUILD.bazel

```diff
@@ -20,18 +21,43 @@ go_library(
     ],
 )
 
-go_binary(
-    name = "bb_copy",
-    embed = [":bb_copy_lib"],
-    pure = "on",
-    visibility = ["//visibility:public"],
-)
+[
+    go_binary(
+        name = "bb_copy-{}".format(goarch),
+        embed = [":bb_copy_lib"],
+        pure = "on",
+        goarch = goarch,
+        goos = "linux",
+        visibility = ["//visibility:public"],
+    )
+    for goarch in ["amd64", "arm64"]
+]
 
-go_image(
+[
```

Comment:
note, there's some code duplication between the three cmd packages. If you'd prefer I can add a macro in container.bzl like `multiarch_go_image`

Reply:
I'd appreciate that. Otherwise we need to patch up this code every time we want to add support for another architecture.